hw/dv/tools/dvsim/xcelium.hjson

```diff
@@ -47,6 +47,10 @@
                "-nowarn SPDUSD",
                // Needed for including "secded_enc.h".
                "-I{build_dir}/src/lowrisc_dv_secded_enc_0",
+               // This warning is thrown when a scalar enum variable is assigned to an enum array.
+               // Other tools (e.g., FPV) treat such assignments as an error, hence we bump it to
+               // an error in simulation so that this can be caught early in CI.
+               "-NCError ENUMERR"
```

Comment:
I suppose you have tested it by intentionally adding this error?

Reply:
Yeah I have added an intentional error in `otp_ctrl` (by removing the curly brackets at one of the lc sync outputs).
Bumping this to an error also uncovered an instance of this issue in DV code (which is fixed in the second commit in this PR).